man/man1/oci-notify.1

```diff
@@ -0,0 +1,93 @@
+.\" Process this file with
+.\" groff -man -Tascii oci-notify.1
+.\"
+.\" Copyright (c) 2020Oracle and/or its affiliates. All rights reserved.
+.\" Licensed under the Universal Permissive License v 1.0 as shown
+.\" at http://oss.oracle.com/licenses/upl.
+.\"
+.TH OCI-NOTIFY 1 "AUGUST 2020" Linux "User Manuals"
+.SH NAME
+oci-notify - Send a message and data to a Oracle Cloud Infrastructure
+notification server.
+.SH SYNOPSIS
+.B oci-notify -c
+.I OCID_TOPIC
+
+.B oci-notify -t
+.I TITLE
+.B -f
+.I FILE
+
+.SH DESCRIPTION
+The
+.B oci-notify
+utility sends a message to an
+.B Oracle Cloud Infrastructure
+notification service. A message is composed of a title and file. Large files
+are split into 64KB chunks. It is the responsibility of the notification service
+configuration where and how the messages are delivered.
+
+.B oci-notify
+command writes log and error messages to the logfile
+.B /var/log/oci-notify.log
```

Comment:
What about having a common approach for all oci-utils utilities ? is it debug messages or messages useful for end users ?